Makefile

```diff
@@ -17,7 +17,7 @@ include common/common.mk
 BUILD_IMAGE=centos:7
 BUILD_TYPE=$(shell ./scripts/deb-or-rpm $(BUILD_IMAGE))
 BUILD_BASE=$(shell ./scripts/determine-base $(BUILD_IMAGE))
-
+PROGRESS=auto
```

Comment:
nit: what about adding some comment here to make it obvious what this option is for? (that also a note for future me 😸 )

Reply:
Had to rebase, so added a comment while at it 👍

Reply:
thx 🤗